### DISCUSSION TITLE
Component Roadmap

### DISCUSSION THREAD

**Discussion by Alex-Stanfield** [heroui-inc/heroui#607] [Feature requests]
Hi there, just joining the group! 

I came across this wonderful library and planning to use in next project(s)

Do we have a component roadmap somewhere?

Besides de missing datepicker (& variants of it) and navbar. Is there something else in the works or any planned dates for them?

Thanks in advance!

Best regards
Alex

PS: Also based in Bs As!

**Comment by jrgarciadev** [comment]
Hey @Alex-Stanfield I'm glad you liked it, the navbar component was released this week https://twitter.com/getnextui/status/1563169730575634433, about the DatePicker we do have it in our roadmap, the roadmap will be public once we release the first stable version

BsAs ❤️
